crawler_mcp/crawlers/optimized/run.py

```
@@ -0,0 +1,1216 @@
+        def _supports_color() -> bool:
+            try:
+                return sys.stdout.isatty() and os.environ.get("NO_COLOR") is None
+            except Exception:
+                return False
+
+        _COLOR = _supports_color()
+
+        def _sty(s: str, code: str) -> str:
+            return f"\033[{code}m{s}\033[0m" if _COLOR else s
+
+        def H(s: str) -> str:  # Header
```

Ruff (0.12.2)

Function name `H` should be lowercase

(N802)